.github/workflows/canary-integration-test.yml

```diff
@@ -303,7 +303,8 @@ jobs:
         run: |
           tests/scripts/github-action-helper.sh create_operator_toolbox
           toolbox=$(kubectl get pod -l app=rook-ceph-tools-operator-image -n rook-ceph -o jsonpath='{.items[*].metadata.name}')
-          timeout 120 sh -c "while kubectl get pod $toolbox -nrook-ceph -o jsonpath='{.items[*].status.phase}'|grep -qE 'Running'; do echo 'waiting for operator toolbox pod to start'; sleep 5; done"
+          # waiting for toolbox operator image pod to get ready
```

Comment:
Please add a commit description about how this improves. Also, was it intentional to reduce the timeout from 120 to 30s? Seems like we should keep the timeout the same.